CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## v2.0.2 - 2020-09-29
 
+## [Unrealeased]
```

Comment:
this should go before the `## v2.0.2` title

Reply:
Right! I have fixed it just now!